_remake/main.js

```diff
@@ -114,27 +123,75 @@ app.use(expressSession({
   }
 }));
 
-
 // flash message middleware
 app.use(flash());
 
+// if app is multi tenant
+// open MySQL connection
+// and use bodyParser (needed for post requests in lib/init-service-routes.js)
+if (RemakeStore.isMultiTenant()) {
+  global.config = {
+    db: {
+      name: "remake-service",
+      port: 3306,
+      host: "localhost",
+      user: "remake",
+      password: "ekamer"
+    },
```

Comment:
Question so if you want to run `multi-tenant` mode you have to setup a SQL server or is this setup by Remake?

Reply:
I think you have to have mysql running in order for multi-tenant to work now. It won't be done by remake.

Reply:
@android4682 for running multi-tenant on a machine MySQL is required. It can run on the same machine or on another machine. Some setup is required - remake won't install things by its own.